### PR TITLE
ADEN-11653 Adding Geo cookie if not exists

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/shared/GeoManager.js [hunk 1]
```diff
@@ -89,6 +90,56 @@ function getGeoDataFromCookie(type = 'country') {
...
+
+            request.send();
+        } catch (err) {
+            console.warn('no geo cookie found');
+            reject();
+        }
+    }).then((geoData) => {
+        Cookies.set('Geo', JSON.stringify(geoData), {
+            domain: getCookieDomain(window.location.hostname),
+            expires: 365,
```
Comment (mfarul) [inline src/shared/GeoManager.js:137]: That's huge, f.e. Varnish set's this as a session cookie. Let's make more like 30 days?

File: src/shared/GeoManager.js [hunk 2]
```diff
@@ -89,6 +90,56 @@ function getGeoDataFromCookie(type = 'country') {
     return false;
 }
 
+export function ensureGeoCookie() {
+    if (Cookies.get(COUNTRY_COOKIE_NAME)) {
+        return Promise.resolve();
+    }
+
+    const GEO_SERVICE_URL = 'https://services.fandom.com/geoip/location';
```
Comment (mfarul) [inline src/shared/GeoManager.js:98]: will this work for dev env?

File: src/shared/GeoManager.js (hunk 1 above)
Comment (dmnsobczak) [reply src/shared/GeoManager.js:137]: In my browser I see that the cookie is set for one year 🤔

File: src/shared/GeoManager.js (hunk 2 above)
Comment (dmnsobczak) [reply src/shared/GeoManager.js:98]: Yes, they both gonna request same endpoint.

**Pull request merged**